### PR TITLE
added: text-utilities and lists

### PR DESCRIPTION
**Pull request by rkapoor10**
Code Snippets will be added later.
text and list

headings

### CODE REVIEW

File: components/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.list-component{
+
+    padding: 1rem 2rem;
+    background-color: var(--white-color);
+}
+
+.indent-x{
+    padding: 0 1rem ;
+}
+
```
Comment (harshitadaswani) [inline components/lists/lists.css]: use utility classes which you have defined

File: components/lists/lists.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                    <p class="txt-m bd-btm mb-2">
+                        Various forms of Lists
+                    </p>
+                </div>
+                <div class="component-display">
+                    <h2 class="mb-1">Spaced (normal lists)</h2>
+                    <div class="display-container flex-icons mb-2">
+                        <div class="list-component ">
+                            <div class="list-title txt-s ">Category</div>
+                            <div class="divider"></div>
```
Comment (harshitadaswani) [inline components/lists/lists.html:73]: instead of using empty div, you can use the class in the upper div

File: components/lists/lists.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                            style="width: 100%; height: 138px;" frameborder="0"></iframe>
+                    </code>
+                </div> -->
+
+                <div class="component-display">
+                    <h2 class="mb-1">Stacked (Notification Page)</h2>
+                    <div class="display-container  mb-2">
+                        <div class="list-component ">
+                            <div class="txt-s ">Category</div>
+                            <div class="divider"></div>
```
Comment (harshitadaswani) [inline components/lists/lists.html:96]: same here

File: components/lists/lists.html [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+                                        <a class="item-hover " href="">Badminton</a>
+                                        <ul class="indent-x ">
+                                            <li class="indent-x item-hover"><a href="">Rackets</a></li>
+                                            <li class="indent-x item-hover"><a href="">Guts</a></li>
+                                            <li class="indent-x item-hover"><a href="">Shoes</a></li>
+                                        </ul>
+                                    </div>
+                                </li>
+                                <li class="txt-s indent-x item-hover gray"><a href="">Training</a></li>
+                                <li class="txt-s indent-x item-hover gray"><a href="">Suppliments</a></li>
```
Comment (harshitadaswani) [inline components/lists/lists.html]: use predefined utility class instead of indent

File: components/lists/lists.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+                    </code>
+                </div> -->
+
+
+                <div class="component-display">
+                    <h2 class="mb-1">List Filters</h2>
+                    <div class="display-container  mb-2">
+                        <div class="list-component ">
+                            <div class="txt-s ">Discount</div>
+                            <div class="divider"></div>
```
Comment (harshitadaswani) [inline components/lists/lists.html:129]: remove empty divs

**Review by harshitadaswani — APPROVED** [review]
Refactor the suggested changes and you are good to go

File: components/lists/lists.css (hunk 1 above)
Comment (rkapoor10) [reply components/lists/lists.css]: done

File: components/lists/lists.html (hunk 2 above)
Comment (rkapoor10) [reply components/lists/lists.html:73]: done

File: components/lists/lists.html (hunk 3 above)
Comment (rkapoor10) [reply components/lists/lists.html:96]: done

File: components/lists/lists.html (hunk 4 above)
Comment (rkapoor10) [reply components/lists/lists.html]: done

File: components/lists/lists.html (hunk 5 above)
Comment (rkapoor10) [reply components/lists/lists.html:129]: done

**Review by Raghav888 — APPROVED** [review]

**Pull request merged**